Assets/Docs/ロコモーションAnimatorController常駐BlendTree化計画書.md

```
@@ -0,0 +1,142 @@
+
+- **途中スポーン時の位相ズレ**: `PlayableAnimationController`のコンストラクタは`_graph.Play()`するだけで、曲の現在時刻への頭出しをしていない。曲の途中でスポーンする敵は、グラフ生成時刻からの経過時間がクリップ時間になるため、**音楽との位相が合っていない**。プレイヤーは曲開始と同時に初期化されるため顕在化していないだけ。
+  - 音楽の現在時刻は`MusicSyncState.PlayTime`（`Assets/Scripts/Runtime/3.Adaptor/InGame/Music/MusicSyncState.cs:12`、double）で取得できる。
+- **Idle↔Walk切り替えの段差**: walkウェイトが速度の生値で、`WALK_THRESHOLD`を跨いだ瞬間に段差が出る（現状は物理側の加減速が実質のスムージング）。BlendTree化に伴いパラメータ側で減衰を掛けられる構造にする。
+
+---
+
+## 3. 設計
+
+### 3.1 グラフ構成（変更後）
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)